### ISSUE TITLE
Broadcast messages in demo viewer

### ISSUE THREAD

**Issue by zooom7z** [teeworlds/teeworlds#2600]
Since my previous issue about kill log is fixed, I have new one.
![screenshot_2020-04-29_00-53-49](https://user-images.githubusercontent.com/35894722/80541758-6469df00-89b4-11ea-87ba-1b15d421ec6a.png)
Settings:
![image](https://user-images.githubusercontent.com/35894722/80542005-e954f880-89b4-11ea-8c09-83445232aa91.png)

When HUD is turned off, join-leave messages is still showing.
Can you turn off these messages? It's annoying when you trying to record demo for movies.

**Comment by Dune-jr** [comment]
Yeah, because those aren't chat messages, they are server messages. This isn't a bug, but should be addressed as part of https://github.com/teeworlds/teeworlds/issues/2293

**Comment by ChillerDragon** [comment]
If it is not about "broadcast" but the message in the chat when players join and leave this is solved by https://github.com/teeworlds/teeworlds/pull/2579

**Comment by ChillerDragon** [comment]
So since #2579 got merged chat can no be fully turned off using ``cl_showchat 0`` I guess this issue can be closed now.

**Comment by oy** [comment]
There should be some setting for it. Best thing would be to do it while doing #2293.
